src/zeek/network.rs

```diff
@@ -1421,3 +1450,96 @@ impl TryFromZeekRecord for Tls {
         ))
     }
 }
+
+impl TryFromZeekRecord for MalformedDns {
```

Comment:
@octoaide
Since Zeek logs do not support `MalformedDns`, please remove the corresponding implementation.